.storybook/themes/news.scss

```diff
@@ -812,6 +812,53 @@
 					list-style-position: inside,
 					list-style-type: disc,
 				),
+				author-bio: (
+					gap: var(--global-spacing-4),
+				),
+				author-bio-author: (
+					gap: var(--global-spacing-4),
+					display: flex,
+					flex-direction: row,
+					components: (
+						image: (
+							max-width: var(--global-spacing-12),
+							max-height: var(--global-spacing-12),
+							border-radius: var(--global-border-radius-2),
+						),
+						stack: (
+							flex: 1,
+						),
+					),
+				),
+				author-bio-social-link-wrapper: (
+					display: flex,
```

Comment:
can this be using the stack component instead of using display flex. Overall, we want to make sure we're using components like Stack where possible to reduce duplication with display: flex. We can set flex-wrap as well based on the components documentation

Reply:
The stack is not flexible for styling, so I don't know if it should always be preferred.  We can discuss this more in depth later, but if some client decided they wanted to change justification, they cannot easily do that.

See: https://washpost.slack.com/archives/C0361V5B7MK/p1661455937171599

Reply:
gap between Description and social-icons is different in desktop and default.

Reply:
The gap should be consistent:
![image](https://user-images.githubusercontent.com/2287238/186779285-01ce5055-2759-4401-8e49-485c78ca2afc.png)
The variance is called out in the full author when I asked @haganka about it:
![image](https://user-images.githubusercontent.com/2287238/186779445-2360e67f-0968-4b0c-a33e-9022ab920508.png)

Reply:
I commented on that here: https://github.com/WPMedia/arc-themes-blocks/pull/1474/files#r955466525

Reply:
huh weird don't see a comment ^

Reply:
it doesn't look like this is a stack component. Not sure if I understand. we don't want to use the stack component?

Reply:
we talked this over in huddle and talked about how this doesn't change direction like the "long" author bio

Reply:
Fixed it in [96f9a90](https://github.com/WPMedia/arc-themes-blocks/pull/1474/commits/96f9a90ab761f67fb76f87c84251162cdb2cc65e)